inference/environments/minimal-ubuntu20.04-py38-cuda11.6.2-gpu-inference/environment.yaml

```diff
@@ -0,0 +1,28 @@
+image:
+  name: azureml/curated/minimal-ubuntu20.04-py38-cuda11.6.2-gpu-inference
```

Comment:
Has this been registered with MCR?

Reply:
Not yet registered with MCR. we will add this name in MCR.

Reply:
We'll need to hold on approving this PR until the image is registered in MCR. Otherwise, it will prevent the release.

Reply:
Also, I'd recommend you use 11.6 here instead of 11.6.2.

Reply:
Ahh, I see it [here](https://github.com/microsoft/mcr/blob/d3d1435bc13f050e5350e6ee59814db522b0d33a/teams/azureml/base/curated-public.yml#L429).